docs/aerodromes/classd/Parafield.md

```diff
@@ -8,9 +8,23 @@
 | Name               | Callsign       | Frequency        | Login ID                         |
 | ------------------ | -------------- | ---------------- | ---------------------------------------- |
 | **Parafield ADC**  | **Parafield Tower**  | **118.700**          | **PF_TWR**                        |
+| Parafield ADC 2†  | Parafield Tower  | 124.600        | PF-2_TWR                        |
 | **Parafield SMC**  | **Parafield Ground** | **119.900**          | **PF_GND**                       |
 | Parafield ATIS        |                | 120.900          | YPPF_ATIS                                |
 
+† *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies){target=new}
+
+### Dual ADC Operations
+When Parafield ADC 2 is online, responsibility for the **Runway**, **Circuit**, and **Airspace** is divided between the two ADC controllers.
+
+ADC 1 takes responsibility for the North/West Runway, Circuit and Airspace (Runway **08L/26R** and **03L/21R**).
```

Comment:
RW ADC 1&2 are reversed... ADC1 owning 21L/26L. Was this deliberate so that a split doesn't steal arrival traffic?

Reply:
Nope not deliberate at all. Had limited info on YPPF operations